iceoryx_hoofs/platform/qnx/include/iceoryx_hoofs/platform/un.hpp

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
```

Comment:
This is weird? Why did you move `wait.hpp` to `un.hpp`?
Where is the original `wait.hpp` file?! Or did here something else go wrong ... I am a little bit lost

Reply:
That's odd, indeed. Shoulnd't be an issue since `wait.hpp` and `un.hpp` are definitely there. I have the feeling I need to practice [this dance](https://www.youtube.com/watch?v=hdcTmpvDO0I) more thoroughly.